xrpl/models/requests/channel_authorize.py

```diff
@@ -50,7 +51,7 @@ class ChannelAuthorize(Request):
     :meta hide-value:
```

Comment:
The class docstrings will also need to be updated (I can't add a comment directly to that part of the file since it's too far away from the lines you're editing). Ditto for all other requests/txs